frontend/src/components/pod/List.tsx

```diff
@@ -21,9 +22,16 @@ export function makePodStatusLabel(pod: Pod) {
 
   return <StatusLabel status={status}>{phase}</StatusLabel>;
 }
+interface PodListProps {
+  pods?: Pod[];
+  error?: ApiError | null;
+}
+export default function PodList(props: PodListProps) {
+  let { pods, error } = props;
+  if (!pods) {
+    [pods, error] = Pod.useList();
```

Comment:
I thought hooks couldn't be used conditionally 🤔

Reply:
Fixed